mne/data/coil_def_Elekta.dat

```diff
@@ -39,6 +39,12 @@
 #			2	accurate
 #
 #
+1   2000    2   1  0.000e+00  0.000e+00	"Point magnetometer, x-normal"
```

Comment:
Currently, 
`FIFF.FIFFV_COIL_POINT_MAGNETOMETER = 2000  # Simple point magnetometer`, which refers to a point magnetometer with a normal vector in the z-direction. I also want to have point magnetometers in the x and y directions. Should I shuffle everything so 
2000 refers to x
2001 refers to y
2002 refers to z?

Or would it be better to append y and z? Leaning towards former option if it won't cause errors.